### PR TITLE
docs: moq docs

### PR DESCRIPTION
**Pull request by bcollamore**

### CODE REVIEW

File: Documentation/Diagnostics/PH2053.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# PH2053:
```
Comment (ynsehoornenborg) [inline Documentation/Diagnostics/PH2053.md:1]: Title missing

File: Documentation/Diagnostics/PH2054.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# PH2054:
```
Comment (ynsehoornenborg) [inline Documentation/Diagnostics/PH2054.md:1]: Title missing

File: Documentation/Diagnostics/PH2054.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+| Diagnostic ID | PH2054 |
+| Category  | [Runtime Failure](../RuntimeFailure.md) |
+| Analyzer | [MockRaiseArgumentsMustMatchEventAnalyzer](https://github.com/philips-software/roslyn-analyzers/blob/master/Philips.CodeAnalysis.MoqAnalyzers/MockRaiseArgumentsMustMatchEventAnalyzer.cs)
+| CodeFix  | No |
+| Severity | Error |
+| Enabled By Default | Yes |
+
+## Introduction
+
+`Mock<T>.Raise(x => x.Event += null, sender, args)` must have correct parameters. There is an argument code mismatch.
```
Comment (ynsehoornenborg) [inline Documentation/Diagnostics/PH2054.md:15]: Duplicate of PH2053 ??

**Review by ynsehoornenborg — CHANGES_REQUESTED** [review]

File: Documentation/Diagnostics/PH2053.md (hunk 1 above)
Comment (bcollamore) [reply Documentation/Diagnostics/PH2053.md]: fixed

File: Documentation/Diagnostics/PH2054.md (hunk 2 above)
Comment (bcollamore) [reply Documentation/Diagnostics/PH2054.md]: fixed

File: Documentation/Diagnostics/PH2054.md (hunk 3 above)
Comment (bcollamore) [reply Documentation/Diagnostics/PH2054.md]: Change "code" to "count".  this is count mismatch. 2053 is type mismatch.

**Review by ynsehoornenborg — APPROVED** [review]

**Pull request merged**